Makefile

```diff
@@ -7,6 +7,9 @@ all:
 	go build -o nspawn-runc ./cmd/nspawn-runc
 	go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
 
+cnibridge:
+	go get -u github.com/containernetworking/plugins/plugins/main/bridge
```

Comment:
As discussed with @nhlfr 
The problem is we strip the vendor folders from our dependencies https://github.com/kinvolk/kubeadm-nspawn/blob/master/Makefile#L15

I am not sure how to handle this... Maybe we could do a glide update/install without the `--strip-vendor` flag first, build the bridge plugin and then do run glide with the flag